source/NVDAObjects/window/winword.py

```diff
@@ -1331,6 +1331,8 @@ def script_lineSpacing(self,gesture):
 		elif val == wdLineSpaceDouble:
 			# Translators: a message when switching to double space  in Microsoft word
 			ui.message(_("Double Spaced"))
+	# Translators: Input help mode text for a command available in Microsoft word.
+	script_reportCurrentComment.__doc__=_("changes the line spacing of the currently selected text.")

```

Comment:
You've put the docstring on the wrong script ;) Also, as this script only makes an existing application command more accessible, the docstring is not as necessary anyway.